auth-server/src/services/auth.ts

```diff
@@ -296,11 +310,14 @@ export default class AuthService {
 				valid: true
 			});
 
-		const user = await getUserFromUserId(userId);
+		user = await getUserFromUserId(userId);
 
 		// send verification email in background
 		sendVerificationEmail(user, verifyToken);
 
+		// send undo token in background
+		sendEmailUndoEmail(user, undoToken);
```

Comment:
Here is another good example where adding "Change" in the name sendEmailUndoEmail**Change** would tremendously help someone not familiar with our code.

Reply:
done